src/app/games/hive/hive-piece.component.ts

```diff
@@ -11,6 +12,10 @@ import { HivePiece } from './HivePiece';
 })
 export class HivePieceComponent extends BaseGameComponent implements AfterContentChecked {
 
+    public constructor(activatedRoute: ActivatedRoute) {
+        super(activatedRoute);
+    }
+
```

Comment:
non ! Pourquoi un sous-composant a besoin d'avoir accès à la route :D

Reply:
Le parent en a besoin, et angular fait l'injection à l'instantiation, on écrit pas nous même this.route = new ActivatedRoute() dans le parents pour pas devoir l'ajouter aux enfants

Reply:
follow-up: pourquoi la classe maman de tous les sous-composant a besoin d'avoir accès au nom du jeu ? (me semble que c'était juste dans le but d'avoir un endroit commun ou extraire la route, mais du coup ce hack vient dégeulasser tous les sous-composants...)